netbox_diode_plugin/api/serializers.py

```diff
@@ -19,10 +19,16 @@
 
 from netbox_diode_plugin.models import Setting
 
-if version.parse(version.parse(settings.VERSION).base_version) >= version.parse("4.1"):
+try:
+    if version.parse(version.parse(settings.VERSION).base_version) >= version.parse(
+        "4.1"
+    ):
+        from core.models import ObjectChange
+    else:
+        from extras.models import ObjectChange
+except version.InvalidVersion as e:
```

Comment:
Please resolve unused `e` local variables as highlighted by ruff linter (here and other two modified files)